src/Passes/README.md

```diff
@@ -1,156 +1,157 @@
-# QIR Passes for LLVM
```

Comment:
What was the reason for removing the general intro and links to the docs? Also the link in ## Out-of-source Pass might be useful to keep.

Reply:
As such we can keep it as training material, but it does not belong at the top-level README as this file should contain the "user" documentation such as "What does the tool do", "How to build the tool", "How to run the tool" and "How do I create new profiles". Passes and the usage of passes  is an implementation detail that only has relevance for those contributing to the implementation of the library.

Reply:
I'd keep the info in the same place as any instructions for contributing to the passes.